stable/traefik/templates/configmap.yaml

```diff
@@ -79,6 +76,10 @@ data:
     {{- end }}
     {{- end }}
     {{- if or .Values.dashboard.enabled .Values.metrics.prometheus.enabled .Values.metrics.statsd.enabled .Values.metrics.datadog.enabled }}
+      {{- if eq .Values.acme.challengeType "dns-01" }}
```

Comment:
@krancour I wonder why `[acme.dnsChallenge]` is conditional on dashboard/prometheus/statsd/datadog being enabled? As a user, I can still want dns challenge for ACME while not enabling dashboard/etc, right? e.g., when traefik is used as an ingress controller.

Reply:
This is a mistake, actually. Nice catch. Fixing.

Reply:
Fixed

Reply:
👍